docs/resources/cluster.md

```diff
@@ -1,7 +1,16 @@
+---
+subcategory: "Deprecated"    
+---
+
+**WARNING:** This resource is deprecated, use `mongodbatlas_advanced_cluster`. In order to learn more about how to migrate, please read the [Migration Guide](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/guides/cluster-to-advanced-cluster-migration-guide).
```

Comment:
Succint phrasing: "To learn more about how to migrate, please read the [Migration Guide](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/guides/cluster-to-advanced-cluster-migration-guide)."

Reply:
thanks, changed